modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductMedia.testcase

```diff
@@ -76,12 +76,80 @@ definition {
 	}
 
 	@description = "This is a test for COMMERCE-6028. AddAnImageToAnOption"
-	@ignore = "true"
 	@priority = "3"
 	test AddAnImageToAnOption {
 		property portal.acceptance = "false";
 
-		// TODO COMMERCE-6028 AddAnImageToAnOption
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Options");
+
+		CPCommerceCatalog.newOptionsEntry(
+			optionDescription = "Option Description",
+			optionFieldType = "Select from List",
+			optionName = "Color");
+
+		Button.clickSave();
+
+		AssertElementPresent(locator1 = "Message#SUCCESS");
+
+		Check.checkToggleSwitch(
+			key_toggleSwitchLabel = "Faceted Navigation",
+			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+		Check.checkToggleSwitch(
+			key_toggleSwitchLabel = "Required",
+			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+		Check.checkToggleSwitch(
+			key_toggleSwitchLabel = "SKU Contributor",
+			locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+
+		CPCommerceCatalog.addOptionValues(
+			optionEntryName = "Color",
+			optionValuesList = "Black:1");
+
+		CommerceNavigator.gotoPortlet(
+			category = "Product Management",
+			portlet = "Catalogs");
+
+		Click(locator1 = "Button#PLUS");
+
+		CommerceEntry.addNewCatalogsEntry(
+			catalogCurrency = "US Dollar",
+			catalogLanguage = "English (United States)",
+			catalogName = "Master");
+
+		AssertClickNoError(
+			locator1 = "Button#SUBMIT",
+			value1 = "Submit");
+
+		Refresh();
```

Comment:
Please remove the creation for the Master catalog because is already created when we do the reindex into the setUp (lines 15-20)